dev_guide/projects.adoc

```diff
@@ -165,3 +165,29 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
+
+You can use node selectors in conjunction with labeled nodes to control pod placement.
+
+[NOTE]
+====
+OpenShift administrators can assign labels link:../install_config/install/advanced_install.html#configuring-node-host-labels[during an advanced installation], or link:../admin_guide/manage_nodes.html#updating-labels-on-nodes[added to a node after installation].
+====
+
+OpenShift cluster administrators can set the cluster-wide default node selector to restrict pod placement to specific nodes. As an OpenShift developer, you can set a node selector on a project configuration to restrict nodes even further. This will be added to the cluster-wide node selector, meaning it will be more restrictive.
+
+To add a node selector, edit a pod configuration, and add the `nodeSelector` value:
+
+====
+----
+...
+nodeSelector:
+  type:user-node,region:east
+...
+----
+====
+
+Any pods created will now be assigned to the labeled nodes.
```

Comment:
No, each pod specification has to include any labels you want on that pod.

"Pods created with a nodeSelector will be assigned to nodes with the specified label(s) and any labels specified for the project by the cluster administrator."